src/file_analysis/analyzer/x509/X509.cc

```diff
@@ -19,9 +19,7 @@
 #include <openssl/opensslconf.h>
 #include <openssl/err.h>
 
-namespace file_analysis {
-std::map<Val*, X509_STORE*> X509::x509_stores;
-}
+#include <iostream>
```

Comment:
That include may be leftover from debugging?

Reply:
indeed :)